x-pack/test/functional/services/transform/edit_flyout.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export function TransformEditFlyoutProvider({ getService }: FtrProviderContext) {
+  const retry = getService('retry');
+  const testSubjects = getService('testSubjects');
+
+  return {
+    async assertTransfromEditFlyoutExists() {
+      await testSubjects.existOrFail('transformEditFlyout');
+    },
+
+    async assertTransfromEditFlyoutMissing() {
+      await testSubjects.missingOrFail('transformEditFlyout');
+    },
+
+    async assertTransformEditFlyoutInputExists(input: string) {
+      await testSubjects.existOrFail(`transformEditFlyout${input}Input`);
+    },
+
+    async assertTransformEditFlyoutInputValue(input: string, expectedValue: string) {
+      const actualValue = await testSubjects.getAttribute(
+        `transformEditFlyout${input}Input`,
+        'value'
+      );
+      expect(actualValue).to.eql(
+        expectedValue,
+        `Transform ${input} input text should be '${expectedValue}' (got '${actualValue}')`
```

Comment:
As well as the suggested edit, should start with `should`

Reply:
I'm not sure I get your suggestion right. How do you think this `expect`message should look like?
We usually have messages in the form of `Expected xyz (got abc)` ([example](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/services/ml/job_wizard_common.ts#L106)) or `xyz should be ... (got abc)` ([example](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/services/ml/data_frame_analytics_creation.ts#L55)).

Reply:
Sorry, you're right @pheyos , I was thinking this was one of test descriptions. Agree we should stick with the same format for these `expect` messages.